docker/run_setup.sh

```diff
@@ -24,3 +24,4 @@ set -e
 
 # Initialize the crontabber bookkeeping for all configured jobs to success
 /app/socorro-cmd crontabber --mark-success=all
+/app/webapp-django/manage.py cron --mark-success=all
```

Comment:
Over time, we're going to rewrite crontabber commands as cron commands. When we do that, the "job" will move from crontabber space to cron space so it'll only ever exist in one world. So we mark everything a success in each world here and that covers everything.